__tests__/main.test.ts

```diff
@@ -50,13 +50,18 @@ describe('run', () => {
   it('(with sync-labels: true) it deletes preexisting PR labels that no longer match the glob pattern', async () => {
     let mockInput = {
       'repo-token': 'foo',
-      'configuration-path': 'bar',
-      'sync-labels': true
+      'configuration-path': 'bar'
     };
 
     jest
       .spyOn(core, 'getInput')
       .mockImplementation((name: string, ...opts) => mockInput[name]);
+    jest
+      .spyOn(core, 'getBooleanInput')
+      .mockImplementation((name: string, opts) => {
+        if (name !== 'sync-labels') throw new Error(`invalid input ${name}`);
+        return true;
+      });
```

Comment:
Unfortunately that doesn't work, because `core.getBooleanInput` doesn't call the mocked version of `core.getInput`.

Reply:
I guess the following code snippet should work fine:
```ts
jest
  .spyOn(core, 'getBooleanInput')
  .mockImplementation((name: string, ...opts) => mockInput[name]);
```

Reply:
Done.

Reply:
That said, the other problem with this approach is it would not catch regressions of the same bug.  I can enhance the test a bit for that, however.

Reply:
I've further updated it to make sure that the `mockInput` is a string-string dictionary so that if one calls `getInput` they'll still get a string, and to make `getBooleanInput` return `=== 'true'` (which is perhaps oversimplified but adequate for purposes of a test mock).